src/function/table.rs

```diff
@@ -179,6 +181,7 @@ impl Table {
                     value: next,
                     timestamp: ts,
                     subsumed: subsumed || prev_subsumed,
+                    cost: cost.or(prev_cost),
```

Comment:
shouldn't we take the smaller cost here?

Reply:
I think just using the new cost also makes sense here... It should stabilize as long as the numbers it's pulling from stabilize. Like in the test that uses this, it uses the `size` function which has its own merge function of smallest, and then updates based on that.

Reply:
To add more context, by leaving the cost as just taking the latest, it would let actions also possibly raise the cost later on. As I said above, it is possible to keep it confluent as long as the sources of costs are also confluent.

Reply:
Updated to use the smaller cost